src/components/common/Coins.js

```diff
@@ -4,11 +4,11 @@ import FontAwesome5Icon from 'react-native-vector-icons/FontAwesome5'
 import useGameContext from 'contexts/Game'
 
 const { width, height } = Dimensions.get('window')
-const FONT_SIZE = width * .07
+const FONT_SIZE = width * .04
```

Comment:
coins component is very small on ipad, maybe we can come up with a better Coins component because the current one isn't sexy, feel free to come up with something 
<img src="https://github.com/Austinae/metrogame/assets/44261381/d11b27cd-ebe7-40b3-9f85-9243e83016de" width="300" />